go.mod

```diff
@@ -34,18 +35,34 @@ require (
 )
 
 require (
+	cloud.google.com/go v0.114.0 // indirect
```

Comment:
so many new deps :(

Reply:
Yeah, I think we find a way around that later